html-syntax/index.html

```diff
@@ -0,0 +1,14 @@
+  <h1>Charmander</h1>
+  <p> &copy Pokemon. &reg Nintendo.</p>
```

Comment:
Make sure you include a semicolon at the end of your HTML entities 🤓

Reply:
Oh I didn't notice that, I'll fix it!